From 5e20f934244244a8ae51657f6efc05e9d7fcab2f Mon Sep 17 00:00:00 2001
From: klzgrad <kizdiv@gmail.com>
Date: Tue, 3 May 2022 01:01:06 +0800
Subject: [PATCH] android: base: Fix build adding extra -latomic

We don't use use_sysroot=true, but without it -latomic is added
to is_android.
---
 src/base/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/base/BUILD.gn b/src/base/BUILD.gn
index afb9ac7de..9e8e05529 100644
--- a/src/base/BUILD.gn
+++ b/src/base/BUILD.gn
@@ -1186,7 +1186,7 @@ component("base") {
   # Needed for <atomic> if using newer C++ library than sysroot, except if
   # building inside the cros_sdk environment - use host_toolchain as a
   # more robust check for this.
-  if (!use_sysroot && (is_android || is_chromeos || (is_linux && !is_castos)) &&
+  if (!use_sysroot && (is_chromeos || (is_linux && !is_castos)) &&
       host_toolchain != "//build/toolchain/cros:host") {
     libs += [ "atomic" ]
   }
-- 
2.53.0

